docs/source/tutorials/8_inverse_kinematics.py

```diff
@@ -0,0 +1,94 @@
+"""=================================================
+Copyright (C) 2025
+License :: Under Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+================================================="""
+
+### Adapted from: https://github.com/kevinzakka/mink/examples
+# REQUIRES:
+# Python 3.9
+# MINK -- pip install "myosuite[examples]"
+
+import os
+
+import mink
+import mujoco
+import mujoco.viewer
+import numpy as np
+from loop_rate_limiters import RateLimiter
+
+os.chdir("../../../myosuite/simhive/myo_sim/arm")
```

Comment:
Is this generic enough? Or something specific?

Reply:
seems to work for now. The issue is always how mujoco handles the relative path. This tutorial is more a proof of concept. It would be good to make it more general